services/alertmanager/alertmanager.go

```diff
@@ -78,7 +82,15 @@ func (svc *Service) Run(ctx context.Context) {
 // TODO That's a temporary measure until we start generating /etc/alertmanager.yml
 // using /srv/alertmanager/alertmanager.base.yml as a base. See supervisord config.
```

Comment:
Comment should be updated: /srv/alertmanager/alertmanager.base.yml generation is a temporary measure, /srv/alertmanager/data creation is not. Maybe that should be two different methods for clarity.

Reply:
Moved to a separate method.